### ISSUE TITLE
Accordion

### ISSUE THREAD

**Issue by pikemurdy** [utkwdn/utkwds#136]
- [x] need an option that is not a header for parent wrapper (button or list or data item)
- [x] Remove margin between each accordion on all variations (on hover the full wrapper should touch the one above)

**Comment by reasterbrooknc** [comment]
Complete, on dev branch.

**Comment by pikemurdy** [comment]
This works. H2 H3 and H4s are now available. I don't see a margin either, but that task is not checked off.

**Comment by spworky** [comment]
Still missing options other than header. I think the original ask was for a <button> or <li> – just a wrapper that would not be a header. <h2> <h3> <h4> have been available in there for awhile.

**Comment by lshoe** [comment]
iirc, we determined that the heading switcher would be adequate for moving forward. It meets the minimum requirement for accessibility. If at some point in the future we decide that a non-heading wrapper is still needed, perhaps we can revisit?

**Comment by lshoe** [comment]
This is another place where some instructional information in the documentation would be helpful—to let users know that the default label is set to H2, and to instruct them on how to locate the switcher.

**Comment by kellieward** [comment]
Looks good
